lib/workflow/client.js

```diff
@@ -103,6 +103,16 @@ class Workflow {
             meta
           })
         });
+      },
+      extend: ({ comment }) => {
+        this.validate({ comment }, 'comment');
+        return this.client(`/${taskId}`, {
+          method: 'PUT',
+          json: this._pack({
+            data: { meta: { extended: true } },
```

Comment:
I wouldn't bother putting this in a `meta` unless you have a strong desire to do so. Adding more layers of `meta` is only likely to cause future us to rage further.